src/response/builder.rs

```diff
@@ -45,6 +45,26 @@ impl ResponseBuilder {
         }
     }
 
+    pub fn headers<I, K, V>(self, headers: I) -> Self
```

Comment:
This will be appreciated by anyone trying to conditionally add a header to a response. I should have included this functionality a long time ago.